data/behavior_data.c

```diff
@@ -1066,7 +1066,7 @@ const BehaviorScript bhvDoor[] = {
     BEGIN(OBJ_LIST_SURFACE),
     SET_INT(oInteractType, INTERACT_DOOR),
     // Door - common:
-    OR_INT(oFlags, (OBJ_FLAG_ACTIVE_FROM_AFAR | OBJ_FLAG_COMPUTE_DIST_TO_MARIO | OBJ_FLAG_SET_FACE_YAW_TO_MOVE_YAW | OBJ_FLAG_UPDATE_GFX_POS_AND_ANGLE)),
+    OR_LONG(oFlags, (OBJ_FLAG_ACTIVE_FROM_AFAR | OBJ_FLAG_COMPUTE_DIST_TO_MARIO | OBJ_FLAG_SET_FACE_YAW_TO_MOVE_YAW | OBJ_FLAG_UPDATE_GFX_POS_AND_ANGLE | OBJ_FLAG_UCODE_LARGE)),
```

Comment:
I almost feel like we should be doing the opposite and have `OBJ_FLAG_UCODE_SMALL`, seems like people are pretty confused about why their objects are clipping. 

Or, what if it depended on the object list? Seems like objects in `OBJ_LIST_SURFACE` shouldn't be rej by default

Reply:
Yeah I agree with this. If there was a flag for both ucodes, since there's a few surface type objects which definitely want to be rejected.

Reply:
yeaaaahhhh maybe OBJ_FLAG_FORCE_REJ and OBJ_FLAG_UCODE_LARGE? or should it be small/large?

Reply:
Yeah, small and large would probably be clearer.